aim/web/ui_v2/src/types/utils/d3/drawHoverAttributes.d.ts

```diff
@@ -6,12 +6,12 @@ import HighlightEnum from 'components/HighlightModesPopover/HighlightEnum';
 
 export interface IDrawHoverAttributesProps {
   index: number;
-  data: IProcessedData[];
   visAreaRef: React.MutableRefObject<>;
   attributesNodeRef: React.MutableRefObject<>;
   attributesRef: React.MutableRefObject<{
     xScale?: IGetAxesScale['xScale'];
     yScale?: IGetAxesScale['yScale'];
+    attributesData: IProcessedData[];
```

Comment:
if we don't need this i think we can remove from here `attributesData: IProcessedData[];`